src/docs/docs/api.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Making an API Public"
+---
+
+## Introduction
+
+In this document, we explain how you make an API public and the considerations you should take into account before you make it public.
+
+## Definitions
+
+### Managed Client
+
+A managed client is one that Sentry controls, such as our frontend. In this type of client, we control both which endpoints are used and the attributes used by those endpoints.
+
+### Partially managed client
```

Comment:
nit: to follow the capitalization of the other headings, this should be Partially Managed Client